README.md

```diff
@@ -1,12 +1,101 @@
 # Genoa
 WIP: Opinionated webhook based GitOps controller that installs and manages helm (v3) releases from configured github/gitlab repositories
 
+Genoa does two important things:
+
+* Allows you to manage your helm releases using a custom resource definitions of the chart name, version, and values
```

Comment:
Perhaps a better way to say: "Allows you to manage your helm releases in a GitOps fashion" ?